src/Lykke.Service.Operations/Workflow/Sagas/BlockchainCashoutSaga.cs

```diff
@@ -138,7 +139,36 @@ public async Task Handle(SolarCashOutCompletedEvent evt, ICommandSender commandS
 
             commandSender.SendCommand(command, "operations");
         }
-        
+
+        #region CashourProcessorEvents
+
+        [UsedImplicitly]
+        public async Task Handle(CashoutsBatchCompletedEvent evt, ICommandSender commandSender)
+        {
+            _log.Info($"CashoutsBatchCompletedEvent for BIL received. BatchId [{evt.BatchId}]", evt);
+
+            if (evt.Cashouts == null || evt.Cashouts.Length == 0)
+            {
+                _log.Warning($"Empty cashouts in batch. BatchId [{evt.BatchId}]", context: evt);
```

Comment:
Better to throw an exception in such case. For instance, contract can be updated and message itself could be correct, but this handler should be altered to handle the message.